src/InfluxDB2/QueryApi.php

```diff
@@ -44,13 +50,20 @@ public function queryRaw(string $query, $org = null, $dialect = null): ?string
     }
 
     /**
-     * @param $query
-     * @param $org
-     * @param $dialect
-     * @return FluxTable[]
+     * Executes the Flux query against the InfluxDB 2.0 and synchronously map the whole response to FluxTable[]
+     * NOTE: This method is not intended for large query results.
+     *
+     * @param string|Query $query
+     * @param string|null $org
+     * @param Dialect|null $dialect
+     * @return  FluxTable[]
      */
-    public function query($query, $org = null, $dialect = null)
+    public function query($query, string $org = null, Dialect $dialect = null): ?array
```

Comment:
Please be consistent. Mark all nullable types with ? or remove ? from ?array.

```suggestion
    public function query($query, ?string $org = null, ?Dialect $dialect = null): ?array
```